database/sqlite/sqlite_functions.c

```diff
@@ -20,6 +20,12 @@ const char *database_config[] = {
     "CREATE TABLE IF NOT EXISTS chart_label(chart_id blob, source_type int, label_key text, "
     "label_value text, date_created int, PRIMARY KEY (chart_id, label_key));",
     "CREATE TABLE IF NOT EXISTS node_instance (host_id blob PRIMARY KEY, claim_id, node_id, date_created);",
+    "CREATE TABLE IF NOT EXISTS alert_hash(hash_id blob PRIMARY KEY, alarm text, template text, "
```

Comment:
This table has only one index, and the rest of the code is not creating indexes for it, on the other hand fields like `plugin`, `module`, and `alarm` are fields we could o queries to show some report from our endpoints, why are we not creating indexes for other fields?

Reply:
@thiagoftsm you are correct -- we may need this in the future but for now it is ok.